tests/views/test_message.py

```diff
@@ -1163,8 +1164,8 @@ def test_reopening_conversation_returns_to_closed_tab(self, mock_request, mock_g
         response_body = response.data.decode("utf-8").replace(" ", "")
 
         # validate that the currently selected tab is as expected (i.e aria-current="location")
-        match_str = '"aria-current="location">Closed'
-        self.assertIn(match_str, response_body)
+        match_str = re.compile(r'"aria-current="location"\s*>Closed<')
```

Comment:
New line characters and whitespace have been introduced so a regex has been used to more accurately match the string.